pkl-codegen-java/src/main/kotlin/org/pkl/codegen/java/JavaCodeGenerator.kt

```diff
@@ -542,7 +541,10 @@ class JavaCodeGenerator(
       val builder =
         TypeSpec.classBuilder(javaPoetClassName.simpleName()).addModifiers(Modifier.PUBLIC)
 
-      if (codegenOptions.implementSerializable && !isModuleClass) {
+      // stateless classes aren't instantiable by choice, i.e., no public ctor is generated
```

Comment:
Comment refers to only one of the applications of `isInstantiable`, located elsewhere. That makes it likely to go out-of-sync. Better to omit.
```suggestion
```